compiler/backend/src/main/core/com/kitakkun/backintime/compiler/backend/valuecontainer/ResolvedValueContainer.kt

```diff
@@ -0,0 +1,174 @@
+        override val getterSymbol: IrSimpleFunctionSymbol,
+        override val serializeAs: IrClassSymbol?,
+    ) : ResolvedValueContainer()
+
+    companion object {
+        context(IrPluginContext)
+        fun create(trackableStateHolder: TrackableStateHolder): ResolvedValueContainer? {
+            val classId = ClassId.fromString(trackableStateHolder.classId)
+
+            val classSymbol = referenceClass(classId) ?: return null
+
+            val allVisibleMemberCallables = classSymbol.owner.getAllMemberCallables().filter { it.owner.visibility.isVisibleOutside() }
+            val allOverriddenCallables = allVisibleMemberCallables.flatMap { it.owner.overriddenSymbols }.toSet()
+            val allPossibleMemberCallables = allVisibleMemberCallables - allOverriddenCallables
+
+            val preSetterFunctionSymbol = when (val preSetter = trackableStateHolder.accessor.preSetter) {
+                is CallableSignature.NamedFunction -> {
+                    allVisibleMemberCallables.find { it.owner.name.asString() == preSetter.name }
+                }
+
+                else -> null
+            }
+
+            val setterFunctionSymbol = when (val setter = trackableStateHolder.accessor.setter) {
+                is CallableSignature.PropertyAccessor.Setter -> allPossibleMemberCallables.find {
+                    it.owner.name.asString() == "<set-${setter.propertyName}>" && it.owner.isSetter
+                }
+
+                is CallableSignature.NamedFunction -> allPossibleMemberCallables.find {
+                    it.owner.name.asString() == setter.name && it.owner.valueParameters.size == 1
+                }
+
+                else -> null
+            } ?: return null
+
+            val captureTargetSymbols = trackableStateHolder.captures.flatMap { captureTarget ->
+                val functionSymbols = when (val signature = captureTarget.signature) {
+                    is CallableSignature.PropertyAccessor.Getter -> {
+                        allVisibleMemberCallables.filter {
+                            it.owner.name.asString() == "<get-${signature.propertyName}>"
+                        }
+                    }
+
+                    is CallableSignature.PropertyAccessor.Setter -> {
+                        allVisibleMemberCallables.filter {
+                            it.owner.name.asString() == "<set-${signature.propertyName}>"
+                        }
+                    }
+
+                    is CallableSignature.NamedFunction.Member -> {
+                        allVisibleMemberCallables.filter { it.owner.matchesFunctionSignature(signature) }
+                    }
+
+                    is CallableSignature.NamedFunction.TopLevel -> {
+                        referenceFunctions(CallableId(FqName(signature.packageFqName), Name.identifier(signature.name))).filter {
+                            it.owner.matchesFunctionSignature(signature)
+                        }
+                    }
```

Comment:
`receiverClassId` is not used. We should use its value properly here.

Reply:
Fixed: [Use receiverClassId for matching logic](https://github.com/kitakkun/back-in-time-plugin/pull/138/commits/3e84dd7f1c0373b42fa8c97fa36969b70dbae9a8)